pkg/istio-agent/xds_proxy.go

```diff
@@ -412,7 +413,7 @@ func (p *XdsProxy) HandleUpstream(ctx context.Context, con *ProxyConnection, xds
 	}
 }
 
-func (p *XdsProxy) handleUpstreamRequest(ctx context.Context, con *ProxyConnection) {
```

Comment:
From the ctx, this thread can know when downstream terminated, how can it know now

Reply:
How is this ctx related to downstream?

Reply:
my mistake, never mind